Orange/widgets/data/oweditdomain.py

```diff
@@ -402,6 +406,10 @@ def __init__(self):
 
         box.layout().addWidget(self.editor_stack)
 
+        self.Warning.add_message(
```

Comment:
I think @janezd didn't like that the change was ignored without a message. I don't see anything about whether the message should be a warning or an error.
We do not want to allow duplicate names (which is not just a cosmetic guideline, it can cause serious problems in the code, because we can index by names etc), so the widget should not output incorrect data. In that case an error is the only appropriate type of message.